draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -269,16 +277,18 @@ is used:
 ".local" or if the value contains more than one ".", then process the candidate
 as defined in {{RFC8445}}.
 
-2. Otherwise, resolve the candidate using mDNS. The ICE agent SHOULD set the
+2. If the ICE agent candidate policy is "relay", as defined in {{JSEP}}, ignore the candidate.
```

Comment:
nit: ICE candidate policy (using the exact term as in JSEP)

Reply:
Fixed